### ISSUE TITLE
Increase Vertical Size of Multi-line Text Input Fields

### ISSUE THREAD

**Issue by slmessier** [kcigeospatial/sha_oed_issues#385]
### Please increase the vertical space within Multi-line Text Input fields listed below to 12 lines:
1. QA-3 request and response page
2. QA-3 response page
3. SWM AS-Built request page
4. SWM AS-Built response page

### Example
(SMW As-Built Response Page)
![image](https://user-images.githubusercontent.com/4098544/36921954-2d34e932-1e34-11e8-95b4-744da9cfcddd.png)

**Comment by vijme** [comment]
IN SWM Request and response page, the allowed text is 500 characters and the box sizes don't match. Can we increase box width in swm request and reduce the height to 4 lines?

**Comment by Tad-Daniel** [comment]
sounds good

maybe make it 6 line to accommodate some possible line breaks.

**Comment by vijme** [comment]
Thank you. Implemented in dev.

**Comment by Tad-Daniel** [comment]
Looks good

**Comment by HollySchuelerKCI** [comment]
This issue has been validated by the project team and confirmed by stakeholders. This issue is ready for staging.

**Comment by HollySchuelerKCI** [comment]
This issue has been validated by the project team in the Staging Environment.

**Comment by Tad-Daniel** [comment]
The interface looks good but I did put in the max number of characters and it appears that the text on the summary sub-tab is not wrapping, the text goes way off screen to the right. It is also happening in the associated email.  I do not recall this from the development env but its happening there also now.

Test in both IE and Edge on win 10

See project AA1545180 in Staging

**Comment by HollySchuelerKCI** [comment]
This issue has been confirmed by the stakeholder and is ready for Production.

**Comment by Tad-Daniel** [comment]
Looks good in production but now the "HELP" button are not there as they are in staging.

**Comment by Tad-Daniel** [comment]
Nope,  My bad the button is there if I put the proper file in the system.  This issue is complete
